services/api-server/src/simcore_service_api_server/services/study_job_models_converters.py

```diff
@@ -87,7 +87,7 @@ async def create_job_outputs_from_project_outputs(
     job_id: StudyID,
     project_outputs: dict[NodeID, dict[str, Any]],
     user_id,
```

Comment:
I guess `mypy` would have capture this if the annotations were in place
Please annotate **all** the inputs, also `user_id`  :-)

Reply:
Ok, I will do that, but in a separate PR. This one should ideally go into staging today to unblock Werner. So I will leave it as is for now.